server/controllers/offer/deleteSavedOffer.js

```diff
@@ -1,3 +1,12 @@
-module.exports = (req, res) => {
-  res.send('delete saved offer ');
+const { deleteSavedOffer } = require('../../database/queries/offers/deleteSavedOffer');
+
+module.exports = (req, res, next) => {
+  const { memberId } = req.params;
+  const { offerId } = req.body;
+  deleteSavedOffer({ memberId, offerId })
```

Comment:
@Fatmasiam It might be a good idea to add validation to ensure that `memberId` and `offerId` aren't undefined.  Also, this is another place where it would be good to use the authenticated memberId in the cookie once that is ready, so that users can't maliciously delete other users' saved offers :)

Reply:
@Fatmasiam please don't forget this